examples/django/django_exampleapp/README.md

```diff
@@ -0,0 +1,40 @@
+# Django Sample Application for Pybrake
+
+## About the application:
+
+The example application provides three GET endpoints:
+
+1. `/date` - returns server date and time. 
+2. `/locations` - returns list of available locations. 
+3. `/weather/{locationName}` - returns the weather details for the locations.
+
+
+## Steps to run the API:
+
+1. Install the dependencies for the application
+
+```bash
+pip3 install -r requirements.txt
+```
+
+2.  Run the localhost server
+
+```bash
+python manage.py runserver 3000 or
+python manage.py runserver 127.10.10.10:3000
```

Comment:
We likely don't need this example. The first one is enough to figure things out.